authentication/templates/registration/base.html

```diff
@@ -0,0 +1,33 @@
+<html lang="en">
```

Comment:
This part is kind of similar to the base template in the `/store` file. You could reuse that.